roles/convert2rhel/tasks/c2r_repos.yml

```diff
@@ -0,0 +1,70 @@
+---
+- name: "Create credentials for 'Convert2RHEL' repository"
```

Comment:
This may not be needed as the CA cert should be preinstalled on RHEL systems. Worth testing though. The less we hardcode the better :-)

Reply:
Pretty sure this is not needed, neither on CentOS nor RHEL.

Reply:
To my surprise it actually doesn't work, after removing this task sync fails with following error:
```
fatal: [localhost]: FAILED! => Task information: 
['Cannot connect to host cdn.redhat.com:443 ssl:default [[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:877)]']
```

Foreman machine:
```
/etc/os-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.9 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.9"
PRETTY_NAME=RHEL
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.9:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.9
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.9"
```